ietf/nomcom/views.py

```diff
@@ -782,12 +782,14 @@ def nominee_staterank(nominee):
 
     sorted_nominees = sorted(nominees,key=lambda x:x.staterank)
 
+    fbtotals = []
```

Comment:
This is a list, but below you treat it as a dict.
What you want here is a defaultdict.
```
>>> from collections import defaultdict
>>> totals=defaultdict(lambda:0)
>>> totals['foo']
0
>>> totals['bar']+=1
>>> totals['bar']
1
```